jquery.leadservice.js

```diff
@@ -221,18 +222,16 @@ define(['jquery', 'jquery.cookie'], function($) {
           data: $(this).serialize(),
           success: function(response) {
             status = 'success';
-            opts.lead_saved();
+            success = opts.lead_saved();
+            if (status == 'success') {
```

Comment:
looks like status is always will be `success`, do we need to keep checking for it?